Update gke-install.md (#719)
entando/entando-docs: vuepress/docs/next/tutorials/getting-started/gke-install.md

:::

### Clustered Storage Using GCP Cloud Filestore

> 1. In the cluster `DETAILS` tab, under `Features` find the `Filestore CSI driver` property and enable it.

Reviewer: ```suggestion
1. In the cluster `DETAILS` tab, under `Features`, find the `Filestore CSI driver` property and enable it
```

> 2. Wait a couple of minutes for the changes to propagate

Reviewer: ```suggestion
2. Wait for the changes to propagate
```

> 3. Ensure also the `Cloud Filestore API` is enabled under `API and services` on the main left menu.

Reviewer: ```suggestion
3. Ensure that the `Cloud Filestore API` is enabled under `API and services` on the main left menu
```

> 3. Verify that your client provisioned successfully. This is indicated by the presence of the storage class with the `rwx` suffix in the output of the following command.

Reviewer: ```suggestion
4. Verify that your client was provisioned successfully. This is indicated by the presence of the storage class with `rwx` suffix in the output following the command.
```

Reviewer: unsure of what was meant by 'the following command'... is the presence of the storage class with the xxx suffix supposed to follow the command? that's what my update suggests

Author: by `the following command` I mean `kubectl get sc` which returns the list of the new generated storage classes

```
NAME                 PROVISIONER                                     RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
NAME                        PROVISIONER                    RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
enterprise-multishare-rwx   filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
enterprise-rwx              filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
premium-rwo                 pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   116m
premium-rwx                 filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
standard                    kubernetes.io/gce-pd           Delete          Immediate              true                   116m
standard-rwo (default)      pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   116m
standard-rwx                filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
```

> 8. Add the variables below to your operator `ConfigMap`

Reviewer: ```suggestion
5. Add the variables below to your operator `ConfigMap`
```

```
entando.k8s.operator.default.clustered.storage.class: "standard-rwx"
entando.k8s.operator.default.non.clustered.storage.class: "standard"
```

> 

Reviewer: the number 9 in line 180 should be '6'
